eclass/cron.eclass

```diff
@@ -131,7 +133,7 @@ docrontab() {
 	exeopts -m0755
 
 	# users expect /usr/bin/crontab to exist...
-	if [[ "${crontab##*/}" != "crontab" ]] ; then
+	if [[ ${crontab##*/} != "crontab" ]] ; then
```

Comment:
remove?

Reply:
Sure, that change by itself is fine, but many such changes increase the risk that there will be some breakage.

I'd rather do only the minimum that is necessary. The rest is not urgent and can be done later (preferably by eclass maintainers if they are still active).